cluster-bootstrap/src/test/scala/akka/management/cluster/bootstrap/internal/BootstrapCoordinatorSpec.scala

```diff
@@ -102,6 +102,59 @@ class BootstrapCoordinatorSpec extends WordSpec with Matchers with BeforeAndAfte
     }
   }
 
+  "BootstrapCoordinator target filtering" should {
+    "not filter when port-name is set" in {
+      val beforeFiltering = List(
+        ResolvedTarget("host1", Some(1), None),
+        ResolvedTarget("host2", Some(2), None),
+        ResolvedTarget("host3", Some(3), None),
+        ResolvedTarget("host4", Some(4), None)
+      )
+
+      BootstrapCoordinator.selectHosts(
+        Lookup.create("service").withPortName("cats"), 8558, filterOnFallbackPort = true, beforeFiltering) shouldEqual beforeFiltering
+    }
+
+    "filter when port-name is not set" in {
+      val beforeFiltering = List(
+        ResolvedTarget("host1", Some(8558), None),
+        ResolvedTarget("host1", Some(2), None),
+        ResolvedTarget("host2", Some(8558), None),
+        ResolvedTarget("host2", Some(4), None)
+      )
+
+      BootstrapCoordinator.selectHosts(
+        Lookup.create("service"), 8558, filterOnFallbackPort = true, beforeFiltering) shouldEqual List(
+        ResolvedTarget("host1", Some(8558), None),
+        ResolvedTarget("host2", Some(8558), None)
+      )
+    }
+
+    "not filter when port-name is not set but filtering disabled" in {
+      val beforeFiltering = List(
+        ResolvedTarget("host1", Some(8558), None),
```

Comment:
I guess this example would be more realistic when it doesn't contain 8558 on both hosts, since in this case ports are arbitrary so would rarely happen to be 8558?

```suggestion
        ResolvedTarget("host1", Some(1), None),
```